shims/spark32/src/main/scala/io/glutenproject/sql/shims/spark32/Spark32Shims.scala

```diff
@@ -64,14 +65,16 @@ class Spark32Shims extends SparkShims {
       readPartitionSchema: StructType,
       options: CaseInsensitiveStringMap,
       partitionFilters: Seq[Expression],
-      dataFilters: Seq[Expression]): TextScan = {
-    new TextScan(
+      dataFilters: Seq[Expression],
+      compressionSplittable: (String) => Boolean): TextScan = {
+    new CHTextScan(
```

Comment:
shim layer is for multiple spark version, don't use to create the specified backend object.

Reply:
`TextScan` is different is spark32/spark33,  and the compressed text format file can not be splittable in clickhouse backend. So I have to define 2 `CHTextScan` class  in shims and override the `isSplittalbe` method to make the flie not to split while read them, or it will throw exception. have any good ways to reslove this?